pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -478,3 +478,15 @@ func CheckNetworkManagerIsRunning() (bool, error) {
 func fixNetworkManagerIsRunning() (bool, error) {
 	return false, fmt.Errorf("NetworkManager is required. Please make sure it is installed and running manually")
 }
+
+func checkIfRunningAsNormalUser() (bool, error) {
+	if os.Geteuid() != 0 {
```

Comment:
This code is duplicated between linux and darwin. Since os.Geteuid() is also available on Windows (it returns -1), this helper function could go to the generic preflight_checks.go, and it would only be called from the linux and darwin files. On Windows, it would be built but not used.

Reply:
This means we have to name the function something different in case of windows, which actually does this check using the powershell package.